tsdb/store.go

```diff
@@ -16,6 +15,8 @@ import (
 	"sync"
 	"time"
 
+	errors3 "github.com/influxdata/influxdb/v2/pkg/errors"
```

Comment:
I thhink `errors3` should be in the block below, not a separate block.

Reply:
It passed the lint stage (and my formatting once I set the right style), so I think this is right

Reply:
Weird.